trin/src/main.rs

```diff
@@ -0,0 +1,11 @@
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    tokio::select! {
+        history = trin_history::main() => {
+            history
+        },
+        state = trin_state::main() => {
+            state
+        },
+    }
```

Comment:
Is there a cleaner way to do this? Do we want any logging here, or should we leave that to the individual network `main()` functions?

Reply:
Yeah, fine to leave it bland for now. Seems okay to me.

At some point we can play with whether we want them in separate threads/runtimes, but for now a 👍🏻 from me.